### ISSUE TITLE
Tab autocomplete in command does not match path with tabedit

### ISSUE THREAD

**Issue by norton120** [VSCodeVim/Vim#4730]
**Describe the bug**
Strangely the tab autocomplete reflects the cwd of the open file, but the filepath it tries to open is the project root. 

**To Reproduce**
with a file tree like:

project
.+ _ _ banana.txt
.+ _ _ inner
....... + - - inner_file.txt
....... + - - other_inner.txt
....... + - - another_inner.txt

If I have `inner_file.txt` open and I type `:tabedit+tab` it will cycle through `inner_file.txt`, `other_inner.txt`, `another_inner.txt` (as expected). if I press `return` when `another_inner.txt` is displayed, I get the following error: 
```
Unable to open 'another_inner.txt': Unable to read file '/Users/myuser/project/another_inner.txt' (Error: Unable to resolve non-existing file '/Users/myuser/project/another_inner.txt').
```
so the path it attempts to open != the path it displays during autocomplete (for tabedit). 

**Expected behavior**
When a file is presented in autocomplete for tabedit it will open in a new tab when I hit enter.

**Environment (please complete the following information):**

- Extension (VsCodeVim) version: 1.13.1
- VSCode version: 1.44.0
- OS: Catalina 10.15.4

**Additional context**
This works fine as far as I can see with `e:`, but not with `:tabedit`.

**Comment by iblislin** [comment]
Hi, I hit this bug as well. Is there a way to make `:tabe` act as `:e` ?

**Comment by eraustud** [comment]
Confirming that this is still a bug with the `:tabe` command. `:e` seems to work as expected, though.

**Comment by iblislin** [comment]
Hi all,
I try to fix it at #9089 .
